devicetypes/Orvibo/Orvibo-Contact-Sensor.src/readme.txt

```diff
@@ -0,0 +1,13 @@
+Easy Setuo Guide:
```

Comment:
File name should be readme.md. Here is guide for creating it: https://guides.github.com/features/mastering-markdown/